tests/test_stepper_motor.py

```diff
@@ -0,0 +1,49 @@
+"""Tests for DummyStepperMotor."""
+import pytest
+
+from finesse.hardware.dummy_stepper_motor import DummyStepperMotor
+
+PRESETS = ("zenith", "nadir", "hot_bb", "cold_bb", "home")
```

Comment:
You can import this from the stepper motor module.

Reply:
Good idea. I've moved it into `config.py` now so that we can also use it from the GUI side and not have to hardcode these values everywhere.